packages/hash/docker/realtime/prod/Dockerfile

```diff
@@ -2,19 +2,24 @@ FROM node:16.14.2-alpine AS builder
 
 WORKDIR /app
 
-COPY packages/hash/api packages/hash/api
-COPY packages/hash/shared packages/hash/shared
-COPY packages/hash/realtime packages/hash/realtime
-COPY packages/hash/backend-utils packages/hash/backend-utils
+# Ensure that the node module layer can be cached
 COPY package.json .
-COPY tsconfig.base.json .
 COPY yarn.lock .
+RUN yarn install --frozen-lockfile --production --ignore-scripts --prefer-offline
+
+# Also ensure that api node modules can be cached
+COPY packages/hash/api/package.json packages/hash/api/
```

Comment:
Not introduced by this PR, but I'm guessing the dependency here comes because `realtime` depends on `backend-utils` which depends on `shared` (for a couple of functions) which depends on generating types from the API schema? or is there something else to it?

`backend-utils` only depends on `shared` for a `sleep` function, so we could remove the need to copy `api` and `shared` in at all if we got rid of that, and keep this image a lot smaller / quicker to build

Reply:
Yes, the dependency here is merely for the dependency on `sleep`. 
I'll make a copy of the sleep function. It is very simple, so I don't think we need to worry about tech-debt with having duplicate code when it's not something that will ever change.